src/test/java/uk/gov/pay/adminusers/service/ServiceInviteCreatorTest.java

```diff
@@ -72,6 +73,30 @@ public void shouldSuccess_serviceInvite_IfEmailDoesNotConflict() {
         assertThat(persistedInviteEntity.getValue().getPassword(), is("encrypted-password"));
     }
 
+    @Test
+    public void shouldSuccess_serviceInvite_IfTelephoneNumberAndPasswordNotPresent() {
+        String email = "email@example.gov.uk";
+        InviteServiceRequest request = new InviteServiceRequest(email);
+        RoleEntity roleEntity = new RoleEntity(Role.role(2, "admin", "Adminstrator"));
+        when(userDao.findByEmail(email)).thenReturn(Optional.empty());
+        when(inviteDao.findByEmail(email)).thenReturn(emptyList());
+        when(roleDao.findByRoleName("admin")).thenReturn(Optional.of(roleEntity));
+        when(notificationService.sendServiceInviteEmail(eq(email), anyString())).thenReturn("done");
```

Comment:
Wow does `notificationService.sendServiceInviteEmail` return a string "done"? Just a comment which doesn't bear on the functionality on this PR.

Reply:
In reality it returns the ID of the notification from Notify.